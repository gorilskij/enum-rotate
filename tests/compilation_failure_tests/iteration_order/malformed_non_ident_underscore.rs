use derive_enum_rotate::EnumRotate;

#[derive(EnumRotate)]
#[iteration_order(_)]
enum Enum {
    A, B, C
}
