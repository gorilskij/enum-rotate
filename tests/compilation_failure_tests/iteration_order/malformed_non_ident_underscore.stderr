error: Expected identifier
 --> tests/compilation_failure_tests/iteration_order/malformed_non_ident_underscore.rs:4:19
  |
4 | #[iteration_order(_)]
  |                   ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/compilation_failure_tests/iteration_order/malformed_non_ident_underscore.rs:7:2
  |
7 | }
  |  ^ consider adding a `main` function to `$DIR/tests/compilation_failure_tests/iteration_order/malformed_non_ident_underscore.rs`
